svsim/src/main/scala/Workspace.scala

```diff
@@ -2,8 +2,8 @@ package svsim
 
 import java.io.{BufferedReader, BufferedWriter, File, FileWriter, InputStreamReader, PrintWriter}
 import java.nio.file.attribute.BasicFileAttributes
-import java.nio.file.{FileVisitResult, FileVisitor}
-import java.nio.file.{FileVisitor, Files, Path, Paths}
+import java.nio.file.{FileVisitResult, FileVisitor, SimpleFileVisitor}
+import java.nio.file.{Files, Path, Paths}
```

Comment:
You could go ahead and merge these two lines. I'm not sure why I didn't have them merged originally...